src/stores/textStore.js

```diff
@@ -0,0 +1,7 @@
+import { defineStore } from 'pinia'
+import {ref} from "vue";
+
+export const textStore = defineStore('texts', () => {
```

Comment:
We have to be careful with store naming. In the Pinia documentation you can read an explanation of why they are named this way: https://pinia.vuejs.org/core-concepts/#Defining-a-Store

```suggestion
export const useTextStore = defineStore('texts', () => {
```